util/secscan/__init__.py

```diff
@@ -86,3 +86,19 @@ def get_priority_for_index(index):
             return priority
 
     return "Unknown"
+
+
+def get_priority_from_cvssscore(score):
```

Comment:
Is there an external reference to these mapping we can refer to?

Reply:
Yes, https://www.first.org/cvss/specification-document#Qualitative-Severity-Rating-Scale
Should I link this to the PR or in the code?

Reply:
Can you add it maybe as a comment, just so it's easier to find for future references.

Reply:
Absolutely.